doc/integrator/internationalization.rst

```diff
@@ -71,6 +70,24 @@ For non Docker project:
      * some metadata as disclaimer
      * editable layer (database structure, data or enumerations)
 
+.. note::
+
+   In mapfiles, attributes added by the mapserver's substitution will be
+   not collected for translation.
+
+~~~~~~~~~~~~~~~~~~~~~~~
+Collecte custom strings
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If the standard system can't collect some strings you can add them manually in
```

Comment:
If the standard system can not collect some strings, you can add them manually in